### PR TITLE
Post on defaulting to Python 3

### PR DESCRIPTION
**Pull request by agjohnson**
We're not yet ready for this to be merged, we have some final pieces and
a release of our docker image to perform before going official with
this.

### CODE REVIEW

File: default-python-3.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.. post:: Jan 11th, 2019
+   :author: Anthony
+
```
Comment (davidfischer) [inline default-python-3.rst]:
```suggestion
   :tags: python

```

By adding the python tag, this ensures that the post gets put into Planet Python's RSS feed.

File: default-python-3.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+======================
+
+Starting February 1st, 2019, new projects getting started with Read the Docs
+will use Python 3 by default. While it is stil possible to configure your
+project to use Python 2.7 with :ref:`our config file <readthedocs:yaml__python>`,
+we think it's important for the ecosystem to help push projects towards adopting
+Python 3.
+
+For the time being, Python 3.6 will be our default Python 3 version, however we
+will have Python 3.7 available for testing very shortly as well.
```
Comment (humitos) [inline default-python-3.rst]: I wouldn't say "for testing" here.

File: default-python-3.rst (hunk 1 above)
Comment (humitos) [reply default-python-3.rst]: I like the idea of "going global" with this post.

**Comment by humitos** [comment]
Linking the PR that contains the docker image modifications for this https://github.com/rtfd/readthedocs-docker-images/pull/84 and the issue onder .org repo defaulting to Python3 https://github.com/rtfd/readthedocs.org/pull/3581

**Comment by humitos** [comment]
Unblocking this since we are close to make Python3 the default. We should take a look at the feedback only.

File: default-python-3.rst (hunk 2 above)
Comment (agjohnson) [reply default-python-3.rst]: We're defaulting to 3.7 now anyways :) i'll update

**Comment by agjohnson** [comment]
I updated this post a bit. In intersphinx links, I noticed our RTD docs have a bad version number, so this might depend on rtfd/readthedocs.org#5259 as well.

**Comment by agjohnson** [comment]
The buttons have been pushed.

**Pull request merged**